pkg/server/backuprestoreserver.go

```diff
@@ -157,6 +157,12 @@ func (b *BackupRestoreServer) runServerWithSnapshotter(ctx context.Context, rest
 
 	go defragmentor.DefragDataPeriodically(ctx, b.config.EtcdConnectionConfig, b.defragmentationSchedule, ssr.TriggerFullSnapshot, b.logger)
 
+	// Lease renewal and lease GC enables only of enable flag set to true. Needed for backward compatability
```

Comment:
Please add this as a `// TODO: @aaronfern ` so that you remember to pick it up later. Just a plain comment might be confusing to read until you've activated the member lease renewal.

Reply:
Done